src/App.jsx

```diff
@@ -14,33 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const IS_SORT_BY_ALPHABET = 'alphabet';
+const IS_SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const getGoods = (goods, sortType, isReversed, isReseted) => {
+  let visibleGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (isReseted) {
+    return visibleGoods;
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (sortType === IS_SORT_BY_ALPHABET) {
+    visibleGoods = visibleGoods.sort((a, b) => a.localeCompare(b));
```

Comment:
```suggestion
    visibleGoods.sort((a, b) => a.localeCompare(b));
```
You don't need to override the variable, just change the array